src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -250,9 +250,11 @@ public RevisionsCollectionConfiguration GetRevisionsConfiguration(string collect
             return Configuration.Default ?? _emptyConfiguration;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This is a big method and not likely to be inlined even with this.
Why is this here?

Reply:
done